openstack/compute/v2/extensions/resetstate/doc.go

```diff
@@ -0,0 +1,11 @@
+/*
+Package resetstate provides functionality to reset the state of a server that have
+been provisioned by the OpenStack Compute service.
+Example to reset a Server
```

Comment:
"reset" should be capitalized as Reset. 

There should also be a newline above and below this line.